pkg/kafsar/offset_helper.go

```diff
@@ -28,6 +28,10 @@ func convOffset(message pulsar.Message, continuousOffset bool) int64 {
 		// TODO get continuous offset
 		return 0
 	}
-	offset, _ := strconv.Atoi(fmt.Sprint(message.ID().LedgerID()) + fmt.Sprint(message.ID().EntryID()) + fmt.Sprint(message.ID().PartitionIdx()))
```

Comment:
please don't extract this method, testcases will not paas when continuous offset enabled